Makefile

```diff
@@ -64,13 +64,13 @@ cs-fix-xml:
 .PHONY: cs-fix-xml
 
 test:
-ifeq ($(shell php --modules|grep --quiet pcov;echo $$?), 0)
-	vendor/bin/simple-phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
-else
 	vendor/bin/simple-phpunit -c phpunit.xml.dist
-endif
 .PHONY: test
 
+coverage:
```

Comment:
test-with-coverage 

?

Reply:
kinda long to write no ?

I also was wondering if I should say something about clover.
Because we might want to run coverage-html locally instead of clover in order to see what is the uncovered code ?